homeassistant/components/zha/core/channels/smartenergy.py

```diff
@@ -154,7 +154,7 @@ def status(self) -> int | None:
         return self.DeviceStatusDefault(status)
 
     @property
-    def unit_of_measurement(self) -> str:
+    def unit_of_measurement(self) -> int:
```

Comment:
The rest of the code checks for 0 zero value, so I'm assuming it's always an int.
Or maybe it can be either a string or an integer?

Reply:
Yeah, i think the value of this cluster is always an `int`